src/app/pages/signing/signing.component.ts

```diff
@@ -16,29 +15,13 @@ import { Fonts } from '../../shared/interfaces';
  * click on the 'Send' button.
  */
 export class SigningComponent implements OnInit {
-  /** Default values */
-  public signingBoxText = '';
-  public backgroundColor = '#ffffff';
-  public textColor = '#000000';
-  public canSend = false;
-  public charCount = 0;
-  public maxCharCount: number;
-  public charCountColor = '#bbbbbb';
-  public name = '[friend name]';
+  public name = 'your friend\'s';
```

Comment:
```suggestion
  public name = 'your friend;
```
Remove the 's since we use the sentence "Type your message to {{ name }} below."